mobile/src/bundles/common/components/button/button.tsx

```diff
@@ -1,19 +1,122 @@
-import React from 'react';
+import { type IconDefinition } from '@fortawesome/free-solid-svg-icons';
+import {
+    type FontAwesomeIconStyle,
+    type Props,
+} from '@fortawesome/react-native-fontawesome';
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
+import React, { useCallback, useState } from 'react';
 import { Pressable } from 'react-native';
+import {
+    type PressableProps,
+    type StyleProp,
+    type ViewStyle,
+} from 'react-native';
 
 import { Text } from '~/bundles/common/components/components';
 
 import { styles } from './styles';
 
 type Properties = {
-    label: string;
-    onPress: () => void;
-};
+    label?: string;
```

Comment:
```suggestion
    text: string;
```

Reply:
we go label property for button in fronedt, quess it should stay label.

Reply:
Ok, but label should be required anyway

Reply:
fix